lib/coverband/utils/results.rb

```diff
@@ -20,6 +20,22 @@ def file_with_type(source_file, results_type)
         get_results(results_type).source_files.find { |file| file.filename == source_file.filename }
       end
 
+      def runtime_relevant_coverage(source_file)
+        eager_loading_coverage = get_results(Coverband::EAGER_TYPE)
+        runtime_coverage = get_results(Coverband::RUNTIME_TYPE)
+        return unless eager_loading_coverage && runtime_coverage
+
+        eager_file = eager_loading_coverage.source_files.find { |file| file.filename == source_file.filename }
+        runtime_file = runtime_coverage.source_files.find { |file| file.filename == source_file.filename }
+
+        return 0.0 unless runtime_file
+
+        return 100.0 unless eager_file
```

Comment:
Is this saying that whenever there is no eager coverage, return 100% runtime coverage? What if all the runtime lines are not hit? I am probably misunderstanding.

Reply:
true this should just be return the normal runtime.percentage not calculating the runtime_relavant_lines

Reply:
ok fixed this and added tests for all 3 cases